sources/packages/backend/libs/sims-db/src/transformers/numeric.transformer.ts

```diff
@@ -14,7 +14,7 @@ export const numericTransformer: ValueTransformer = {
    * @returns numeric/decimal string converted to a number.
    */
   from: (value: string | null): number | null => {
-    if (value) {
+    if (value !== null && value !== undefined) {
```

Comment:
Suggestion from my side to use the following solution. 

```ts
from: (value: string | null): number | null => {
    if (Number.isFinite(parseFloat(value))) {
      return parseFloat(value);
    }
    return null;
  }
```

Reply:
Thanks for suggesting it @dheepak-aot and thanks for implementing it @sh16011993.